src/headerssync.cpp

```diff
@@ -0,0 +1,312 @@
+#include <headerssync.h>
```

Comment:
This file is missing the copyright notice.

Reply:
Has been added.